### PR TITLE
Add support for Razer Kraken Kitty Edition

### PR DESCRIPTION
**Pull request by 999eagle**
This adds support for the Kraken Kitty Edition headset (see #949).

As the Kraken Kitty Edition for *some* reason uses the Chroma protocol instead of the protocol all other Kraken headsets use, I've decided to put the driver into a new kernel module called `razerkrakenchroma` to keep the code for the `razerkraken` module clean.

The headset supports various `chroma_extended_matrix` effects even though none of the effects are supported in Synapse.

### CODE REVIEW

**Comment by z3ntu** [comment]
Thanks for the PR! Please put the device into the razeraccessory driver though to avoid code duplication.

**Comment by 999eagle** [comment]
Okay, I've moved everything into `razeraccessory`‌ now.

File: pylib/openrazer/_fake_driver/razerkrakenkittyedition.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[device]
+dir_name = 0003:1532:0f19.0001
+name = Razer Kraken Kitty Chroma Control
```
Comment (z3ntu) [inline pylib/openrazer/_fake_driver/razerkrakenkittyedition.cfg]: should be `Razer Kraken Kitty Edition`

File: daemon/openrazer_daemon/hardware/headsets.py [hunk 2]
```diff
@@ -303,3 +303,23 @@ def _resume_device(self):
...
+    USB_PID = 0x0F19
+    METHODS = ['get_device_type_headset',
+               'set_none_effect', 'set_static_effect', 'set_breath_random_effect', 'set_breath_single_effect',
+               'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row',
+               'set_brightness', 'get_brightness']
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 4]
+
+    DEVICE_IMAGE = "https://assets3.razerzone.com/C2Ol5yRpYX8kE8JX3R9K2svFNaM=/1500x1000/https%3A%2F%2Fhybrismediaprod.blob.core.windows.net%2Fsys-master-phoenix-images-container%2Fhda%2Fh57%2F9055450103838%2FKraken-Kitty-Edition-Black-gallery-Hero.jpg"
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/headsets.py]: Maybe use https://hybrismediaprod.blob.core.windows.net/sys-master-phoenix-images-container/hda/h57/9055450103838/Kraken-Kitty-Edition-Black-gallery-Hero.jpg directly; it's interesting how many different subdomains and media hosts Razer uses

File: daemon/openrazer_daemon/hardware/headsets.py [hunk 3]
```diff
@@ -303,3 +303,23 @@ def _resume_device(self):
...
 
         self.disable_notify = False
+
+
+class RazerKrakenKittyEdition(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Kraken Kitty Edition
+    """
+    EVENT_FILE_REGEX = re.compile(
+        r'.*Razer_Kraken_Kitty_Chroma_Control-event-if00')
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/headsets.py]: without the line break please

**Review by z3ntu — CHANGES_REQUESTED** [review]
just a few small things, looks good otherwise

**Comment by tbaumann** [comment]
Tested and works :+1:

**Review by z3ntu — APPROVED** [review]
LGTM, thanks a lot!

@tbaumann Thank you for testing!

**Comment by Kusoneko** [comment]
Hello,

I would like to ask if any of you probed the device for support of the other Chroma effects besides None, Static and Breath (Single, Dual & Random) when adding support for the device, as I'm currently trying to figure out if it's possible to add the other effects or if I'm wasting my time trying to add support for them to the driver of this device.

Thank you,
Kusoneko

**Comment by z3ntu** [comment]
@Kusoneko There's a PR for that now: #1528

**Pull request merged**